README.md

```diff
@@ -28,9 +28,9 @@ https://groups.google.com/a/arduino.cc/forum/#!forum/developers
 
 Installation
 ------------
-Detailed instructions for installation in popular operating systems.  
```

Comment:
The trailing whitespace on this line as well as 32 and 33 was intentional but this could be avoided (and the text improved) by formatting this section as a list.